build.sbt

```diff
@@ -4,6 +4,16 @@ organization in ThisBuild := "me.shadaj"
 
 scalaVersion in ThisBuild := "2.12.8"
 
+Compile / scalaSource := baseDirectory.value / "src" /  "main" / "scala"
+
+Test / scalaSource := baseDirectory.value / "src" / "test" / "scala"
+
+// dependencies for scalapy-tensorflow-source
+libraryDependencies += "me.shadaj" %%% "scalapy-core" % "0.3.0"
+libraryDependencies += "me.shadaj" %%% "scalapy-numpy" % "0.1.0+3-046d1d67"
+libraryDependencies += "org.scalatest" %%% "scalatest" % "3.1.0-SNAP8" % Test
```

Comment:
Hmm why use snapshot release? A regular scalatest 3.1.0 is already released

Reply:
well those are production dependencies of this release, copied form `scalaPyTensorFlowCross`

Reply:
Can we upgrade to `3.1.2`?